pages/about.html

```diff
@@ -44,26 +44,31 @@
         <section class="home-section about-section">
```

Comment:
- [ ] [OPTIONAL] Kinflr make sure to fix the `Home` page link in the nav bar on the about page it looks like this for now `  <li class="nav-link-list"><a href="/serategna">Home</a></li>`, you can change ` href="/serategna` to `href="/"` instead so the link redirects to the Home page

![Screenshot_20230601_100137](https://github.com/habitoz/serategna/assets/25227022/ad4b4a3d-696b-4065-99fa-fb4488916c4a)